pkg/validators/privileges/role_privileges.go

```diff
@@ -49,11 +49,11 @@ func (s *PrivilegeValidationService) ReconcileRolePrivilegesRule(rule v1alpha1.G
 	defer cancel()
 
 	vr := buildValidationResult(rule, constants.ValidationTypeRolePrivileges)
-	failMsg := fmt.Sprintf("One or more required privileges was not found, or a condition was not met for account: %s", rule.Username)
+	failMsg := fmt.Sprintf("One or more required privileges was not found, or a condition was not met for account: %s", rule.Name())
```

Comment:
imo its more intuitive that this is not changed.
its not intuitively obvious that the rule name is the username.

Reply:
Good catch. I was assuming that every time we accessed the field of the rule that happened to be how we generated the rule name, it because we wanted to access the name. I should have read this line more carefully.

Reply:
no worries, its a pretty easy thing to miss when youre making 6 PRs which are all pretty much the same thing